### DISCUSSION TITLE
Party Plugin : Bring back the moveable interface overlay with a toggle

### DISCUSSION THREAD

**Discussion by 1Piwix0** [runelite/runelite#15131] [Ideas]
![image](https://user-images.githubusercontent.com/82069937/174400065-68e56fe3-2465-4de8-81aa-01627a79644e.png)
Original party plugin user here, I would like to see an option for the original display to return as a toggle. I'd like to be able to move the party's health/prayer info closer to my field of focus in some instances and still see them while having my sidebar collapsed

**Comment by leadanymind** [comment]
I would also like to see the spec energy as well. While we're talking about the party plugin, it'd be great if we could get chat box messages for things like hitting unpotted, or hitting an enemy without offensive prayers. Might be worth a separate issue/task, but is along the same vein of conversation.
